src/views/Feed.vue

```diff
@@ -1,12 +1,11 @@
 <template>
   <ion-page>
+    <ion-button class="edit-button" v-if="canPost" @click="createPost(canPostInTeams)" shape="round" size="small">
+       <ion-icon size="small" :icon="editIcon"/>
+    </ion-button>
+    
     <ion-content data-cy="activity-feed">
       <div class="wrap">
-        <ion-card v-if="canPost">
-          <ion-card-content>
-            <new-post :teams="canPostInTeams" />
-          </ion-card-content>
-        </ion-card>
```

Comment:
You are removing the new-posting-widget entirely, but we still want this to be existing as before on large-widescreen-views (e.g. Desktop or iPad landscape). Ideally we should add a test that ensure it is visible on those and not on mobile.